src/main/kotlin/graphics/scenery/backends/opengl/OpenGLRenderer.kt

```diff
@@ -2213,11 +2213,16 @@ open class OpenGLRenderer(hub: Hub,
             screenshotRequested = false
         }
 
+        postRenderLambdas.forEach {
+            it.invoke()
+        }
+
         stats?.add("Renderer.${flow.last()}.renderTiming", System.nanoTime() - startPass)
 
         updateLatch?.countDown()
         firstImageReady = true
         frames++
+        totalFrames++
```

Comment:
@skalarproduktraum I have generalized the totalFrames parameter to the OpenGLRenderer, as was your suggestion. Is the OpenGLRenderer recreated at any point, which would then require totalFrames to be set to 0 again? I could not find any such recreation

Reply:
No, no such things happens for OpenGL 👍